packages/babel-parser/src/plugins/flow/index.ts

```diff
@@ -2031,7 +2020,7 @@ export default (superClass: typeof Parser) =>
 
     parseConditional(
       expr: N.Expression,
-      startPos: number,
+
```

Comment:
Hey prettier, were you sleeping? 😅

Reply:
https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0AEAhgBQA6UhANOQEbVTlj0AmAlMOfvp91AL4hKICAAdc0AM7JQhAE6yIAdwAKchFJSEANosIBPKUJqzCYANZwYAZUIBbOABkcUOMlTaJcIyfOWrI02cAc2QYWXQvEDhbGjhmZjiHQigg9EIguAAxCFlbQhhcFOQQQnQYCEEQAAsYWy0AdSqceAkAsDgrdWacADdmvWKwCUMQZ09ZGGUTILy3D0iAKwkADytgrTgARXQIeDmtTyEA2XHimkJYrUqRWWcYepxmGCrkAA4ABiOFT3qTEWKbnBxj1XEIAI47eBTUQaEoSAC0LjicUqsjgEJwaKm6VmSHcB0inlsOFC4UJ6y2kNcePmQhgFweTxeSAATHSTDgtMEAMIQWy4qISACslXQngAKhcNPjDiAehEAJJQBKwKxgW5iACCyqsMD0G32nj4fCAA
This is expected, prettier allows to keep at most one blank line.